contracts/smart-contract-wallet/SmartAccount.sol

```diff
@@ -265,7 +265,8 @@ contract SmartAccount is
             userOp.signature,
             (bytes, address)
         );
-        // TODO: USE if(isModuleEnabled()) instead?
+        // TODO:
+        // 1) USE if(isModuleEnabled()) instead?
```

Comment:
what's the benefit? should we do this right away?

Reply:
isModule enabled has an extra check `SENTINEL_MODULES != module`
it's more secure, but consumes more gas
however, in my tests trying to call SENTINEL_MODULES.validateUserOp() reverts. so it's safe to not check for it. However, I'm interested to hear Zellic opinion on it. Maybe they can figure out a scenario when SENTINEL_MODULES.validateUserOp() doesn't revert.